design/release-notes/release-0.9/draft-release-notes.md

```diff
@@ -0,0 +1,119 @@
+## Notable Items
+### New CertificateRequest Resource
+A new resource has been introduced - `CertificateRequest` - that is used to
+request certificates using a raw x509 certificate signing request. This resource
+is not typically used by humans but rather by other controllers or services. For
+example, the `Certificate` controller will now create a `CertificateRequest`
+resource to resolve its own Spec.
+
+Controllers to resolve `CertificateRequest`s are currently disabled by default
+and enabled via the feature gate `CertificateRequestControllers`. This feature
+is currently in Alpha and only the CA issuer has been implemented.
+
+This resource is going to enable out of tree, external issuer controllers to
+resolve requests. Other issuer implementations and details on how to develop an
+out of tree issuer will follow in later releases. You can read more on the
+motivations and road map in the [enhancement
+proposal](https://github.com/jetstack/cert-manager/blob/master/design/20190708.certificate-request-crd.md)
+or how this resource is used in the
+[docs](https://docs.cert-manager.io/en/release-0.9/reference/certificaterequests.html).
+
+
+### DNS Zones support for ACME challenge solver selector
+A list of DNS zones can now be added to the ACME challenge solver selector.  The
+most specific DNS zone match specified here will take precedence over other DNS
+zone matches, so a solver specifying `sys.example.com will` be selected over one
+specifying `example.com` for the domain `www.sys.example.com`. If multiple
+solvers match with the same dnsZones value, the solver with the most matching
+labels in matchLabels will be selected. If neither has more matches, the solver
+defined earlier in the list will be selected.
+
+### Certificate Readiness Prometheus Metrics
+Cert-manager now exposes Prometheus metrics on Certificate ready statuses as
+`certmanager_certificate_ready_status`. This is useful for monitoring
+Certificate resources to ensure they have a `Read=True` status.
+
+### Prometheus Operator ServiceMonitor
+Support has been added to include a Prometheus ServiceMonitor for cert-manager
+in the helm chart. This enables monitoring of cert-manager when in conjunction
+with the Prometheus Operator. This is disabled by default but can be enabled via
+the helm configuration.
+
+### ACMEv2 POST-as-GET
+
+- Add support for ACMEv2 POST-as-GET ([#1648](https://github.com/jetstack/cert-manager/pull/1648), [@munnerz](https://github.com/munnerz)) <- this one means we are now ACMEv2 spec compliant
+
+### ACME Issuer Solver Pod Template
+The ACME Solver Pod Spec now exposes a template that can be used to change
+metadata about that pod. Currently, a template will expose labels, annotations,
+node selector, tolerations, and affinity.
```

Comment:
Add `This is useful when running cert-manager in multi-arch clusters, or when you run workloads across different types of nodes and need to restrict where the acmesolver pod runs.`